.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Django: manage.py runserver",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/website/manage.py",
+            "args": [
+                "runserver",
+            ],
+            "justMyCode": true,
+            "django": true,
+        },
```

Comment:
Maybe add a test runner too? That way people can run tests in one click, and it also works with the graphical debugger.

Reply:
As of now, integrating with the VSCode testing tab (so people can e.g. run single tests from the UI) is not possible. But some work is going on: https://github.com/microsoft/vscode-python/pull/21468 is under consideration. 

I can add a launch config to simply run the tests with a debugger, but 'breakpoint on uncaught exception' will not open automatically open the debugger when a test fails/errors. But it's useful if you manually set breakpoints of course. Sadly a launch config also doesn't really make it easy to filter the tests you want to run.